blog/2022/02/10years.md

```diff
@@ -0,0 +1,14 @@
+@def rss_pubdate = Date(2021, 2, 14)
+@def rss = """10 years ago today, we published "Why we Created Julia" ..."
+@def published = "14 February 2021"
+@def title = "Why We Use Julia 10 years on"
```

Comment:
```suggestion
@def title = "Why We Use Julia, 10 Years Later"
```

I could be wrong but I think we should use later here: https://ell.stackexchange.com/questions/258105/four-years-on-vs-four-years-later